### ISSUE TITLE
Missing PNG's in artwork repository

### ISSUE THREAD

**Issue by frantechca** [turnkeylinux/tracker#721]
Hello,

It seems there's a lot of art work missing. So far it seems to be:

- ghost
- mattermost
- mediaserver
- odoo

Much thanks,

Francisco

**Comment by frantechca** [comment]
It also seems the PNG for EspoCRM isn't on a transparent background.

Francisco

**Comment by JedMeister** [comment]
Sorry, that's my fault. I dropped the ball on updating the artwork directory when I've added new appliances. FWIW they should all be in the `.art` directory of each appliance. E.g. [Mattermost](https://github.com/turnkeylinux-apps/mattermost/tree/master/.art)

Upstream developed the EspoCRM icon and I missed that it doesn't have transparent background. We'll need to fix that...

I'll try to tidy this up for v14.2 but feel free to do some tidying up if you have the time or the inclination.
